### DISCUSSION TITLE
Documentation of file location of Step 5 Bare Metal Installation

### DISCUSSION THREAD

**Discussion by smilebasti** [deckhouse/deckhouse#175] [Q&A] [answered]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Version

v1.0.0

### Expected Behavior

I just installed deckhouse following this: https://deckhouse.io/en/gs/bm/step5.html

But i could find where to put the files: nginx-ingress-controller.yml and user.yml

### Actual Behavior

Would be great to find this in that description as it is easier to follow.

Thanks ahead for this short answer

**Comment by nabokihms** [comment, marked as answer]
@smilebasti hello! Thanks for the question.

You need to:
1. Connect to the first control plane node via SSH
2. Login as the root user 
3. Save files to tmp
4. Apply them using kubectl like this:
    ```shell
    kubectl create -f /tmp/nginx-ingress-controller.yml
    ```

**Reply by nabokihms** [reply]
We will add a short description to the doc.

**Reply by smilebasti** [reply]
Thanks

**Reply by nabokihms** [reply]
https://github.com/deckhouse/deckhouse/commit/431d1f0ac8ee95d5ff96614c0054285fdc1035d9
